### PR TITLE
MAT-903 and MAT-904 harp user emails

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/resources/templates/mail/welcomeTemplate.ftl [hunk 1]
```diff
@@ -3,11 +3,13 @@
 	<meta http-equiv="Content-type" content="text/html;charset=UTF-8">
 </head>
 <body>
-<p style="font-family: 'Tahoma'; font-size: 10pt;">You have been granted access to the Measure Authoring Tool. Your User ID is ${(loginId)!}. Your temporary password will be sent in a separate email. This password expires in 5 days. To access the Measure Authoring Tool, please go to www.emeasuretool.cms.gov. 
+<p style="font-family: 'Tahoma'; font-size: 10pt;">You have been granted access to the Measure Authoring Tool. Your can use your HARP ID ${(harpId)!} to access the Measure Authoring Tool at ${(url)!}.
```
Comment (RohitKandimalla) [inline src/main/resources/templates/mail/welcomeTemplate.ftl:6]: **You** can user your HARP ID. Typo

Comment (serhii-ilin) [reply src/main/resources/templates/mail/welcomeTemplate.ftl:6]: wow! good catch! thanks

File: src/main/resources/templates/mail/tempPasswordTemplate.ftl [hunk 2]
```diff
@@ -1,6 +1,7 @@
-You have been assigned a temporary password to the Measure Authoring Tool.  Your temporary password is ${(password)!}
-This password expires ${(passwordExpireDate)!} at 12:00 AM, Central Time.
+Your account has been re-activated. Your can use your HARP ID ${(harpId)!} to access the Measure Authoring Tool at ${(url)!}
```
Comment (RohitKandimalla) [inline src/main/resources/templates/mail/tempPasswordTemplate.ftl:1]: Same Typo error

**Review by RohitKandimalla — APPROVED** [review]

File: src/main/resources/templates/mail/tempPasswordTemplate.ftl (hunk 2 above)
Comment (serhii-ilin) [reply src/main/resources/templates/mail/tempPasswordTemplate.ftl:1]: wow!

File: src/main/resources/templates/mail/welcomeTemplate.ftl (hunk 1 above)
Comment (serhii-ilin) [reply src/main/resources/templates/mail/welcomeTemplate.ftl]: fixed

**Review by adongare — APPROVED** [review]

**Pull request merged**